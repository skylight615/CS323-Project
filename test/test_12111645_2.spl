int test_2(){
    int a=0X0;
    int b=0x12ab;
    int d=4294967295;

}