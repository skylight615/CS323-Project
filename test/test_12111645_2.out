Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: test_2
        LP
        RP
      CompSt (1)
        LC
        BodyList (2)
          DefList (2)
            Def (2)
              Specifier (2)
                TYPE: int
              DecList (2)
                Dec (2)
                  VarDec (2)
                    ID: a
                  ASSIGN
                  Exp (2)
                    INT: 0X0
              SEMI
            DefList (3)
              Def (3)
                Specifier (3)
                  TYPE: int
                DecList (3)
                  Dec (3)
                    VarDec (3)
                      ID: b
                    ASSIGN
                    Exp (3)
                      INT: 0x12ab
                SEMI
              DefList (4)
                Def (4)
                  Specifier (4)
                    TYPE: int
                  DecList (4)
                    Dec (4)
                      VarDec (4)
                        ID: d
                      ASSIGN
                      Exp (4)
                        INT: 4294967295
                  SEMI
        RC
